nautobot/ipam/forms.py

```diff
@@ -323,6 +321,16 @@ class Meta:
             "date_allocated": DateTimePicker(),
         }
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        if self.instance is not None:
+            self.initial["vrfs"] = self.instance.vrfs.all().values_list("id", flat=True)
```

Comment:
Not that it really matters, but this is a thing:

```suggestion
            self.initial["vrfs"] = self.instance.vrfs.values_list("id", flat=True)
```